### PR TITLE
Fixes new api

### PR DESCRIPTION
**Pull request by yxdunc**

### CODE REVIEW

File: snipsmanagercore/server.py [hunk 1]
```diff
@@ -49,17 +63,12 @@ def __init__(self,
...
-        else:
-            self.tts_service = SnipsTTS(
-                self.thread_handler,
-                mqtt_hostname,
-                mqtt_port,
-                "hermes/tts/say",
-                locale,
-                logger=self.logger)
+        if tts_service_id not in ["snips", "google", None]:
+            self.log_error("Warning ! We only support Snips or Google TTS.")
```
Comment (michaelfester) [inline snipsmanagercore/server.py]: Do we need to show an error here? Fine to create assistants without TTS.

Also, no space before "!" in English :)

**Review by nebuto — APPROVED** [review]
LGTM

File: snipsmanagercore/snips_dialogue_api.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                type(tts_service_id) is str and tts_service_id.decode('utf-8').lower() == "snips"):
+            self.tts_method = self.end_session
+        elif (type(tts_service_id) is str and tts_service_id.decode('utf-8').lower() == "google"):
+            self.tts_method = self.google_end_session
+
+    def speak(self, ttsContent, sessionId=None):
+        if (self.tts_method is not None):
+            self.tts_method(ttsContent, sessionId)
+
+    def google_end_session(self, ttsContent, sessionId=None):
```
Comment (michaelfester) [inline snipsmanagercore/snips_dialogue_api.py]: Use underscore instead of camelCase for variable names

File: snipsmanagercore/server.py (hunk 1 above)
Comment (yxdunc) [reply snipsmanagercore/server.py]: No need for this warning anymore! :)

**Pull request merged**